Multisig/UI/ClaimToken/Select Guardian/SelectGuardianViewController.swift

```diff
@@ -49,12 +48,11 @@ class SelectGuardianViewController: ContainerViewController, UISearchBarDelegate
 
         keyboardBehavior = KeyboardAvoidingBehavior(scrollView: guardiansController.tableView)
         keyboardBehavior.hidesKeyboardOnTap = false
-        navigationController?.navigationBar.prefersLargeTitles = true
         searchBar.searchTextField.delegate = self
     }
 
     func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
-        guardiansController.filterData(searchTerm: searchText)
+//        guardiansController.filterData(searchTerm: searchText)
```

Comment:
commented out code: uncomment or remove?

Reply:
this whole class will be deleted in another pr